src/libraries/Common/src/Interop/Windows/Kernel32/Interop.UNICODE_STRING.cs

```diff
@@ -0,0 +1,27 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Kernel32
+    {
+        // https://msdn.microsoft.com/en-us/library/windows/desktop/aa380518.aspx
+        // https://msdn.microsoft.com/en-us/library/windows/hardware/ff564879.aspx
+        [StructLayout(LayoutKind.Sequential, CharSet = CharSet.Unicode)]
+        internal unsafe struct UNICODE_STRING
```

Comment:
This exists as `src\libraries\common\src\Interop\Windows\Interop.UNICODE_STRING.cs` already